docs/cpp/customize-default-settings-cpp.md

```diff
@@ -0,0 +1,109 @@
+---
+Order: 7
+Area: cpp
+TOCTitle: Customize default settings
+ContentId: 4E34F6AF-BFC6-4BBB-8464-2E50C85AE826
+PageTitle: Customize default settings in Visual Studio Code C++ projects
+DateApproved: 06/25/2019
+MetaDescription: How to customize semantic colorization of C++ code in Visual Studio Code.
+---
+
+# Customizing Default Settings
+
+You can override the default values for properties set in `c_cpp_properties.json`.
+
+## New VS Code settings
+
+The following `C_Cpp.default.*` settings map to each of the properties in a configuration block of `c_cpp_properties.json`. Namely:
+
+```json
+C_Cpp.default.includePath                          : string[]
+C_Cpp.default.defines                              : string[]
+C_Cpp.default.compileCommands                      : string
+C_Cpp.default.macFrameworkPath                     : string[]
+C_Cpp.default.forcedIncludes                       : string[]
+C_Cpp.default.intelliSenseMode                     : string
+C_Cpp.default.compilerPath                         : string
+C_Cpp.default.cStandard                            : c89 | c99 | c11
+C_Cpp.default.cppStandard                          : c++98 | c++03 | c++11 | c++14 | c++17
```

Comment:
There are more settings available since the time of writing.  Please work with my team to get the updated list.